pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -61,6 +61,7 @@ def backward(
         # unscale gradient to allow analyze within `on_after_backward`
         if not should_accumulate and model.automatic_optimization:
             self.scaler.unscale_(optimizer)
+            self.move_grad_to_cpu(model.trainer.model)
```

Comment:
This sounds like we always do this. can we either put the if hasattr(model, "cpu_offload") here or find a better name for the function?

Reply:
Or maybe we should do this per default?